docs/book/toc.md

```diff
@@ -8,15 +8,20 @@
 
 ## Guides
 
-* [Functional API](guides/low-level-api/README.md)
-  * [Create a step](guides/low-level-api/chapter-1.md)
-  * [Normalize data](guides/low-level-api/chapter-2.md)
-  * [Train & Evaluate](guides/low-level-api/chapter-3.md)
-  * [Caching](guides/low-level-api/chapter-4.md)
-  * [Materialize artifacts](guides/low-level-api/chapter-5.md)
-  * [Import dynamic data](guides/low-level-api/chapter-6.md)
-  * [Deploy to production](guides/low-level-api/chapter-7.md)
-* [Class based API](guides/high-level-api/README.md)
+* [Functional API](guides/functional-api/README.md)
+  * [Create a step](guides/functional-api/chapter-1.md)
+  * [Normalize data](guides/functional-api/chapter-2.md)
+  * [Train & Evaluate](guides/functional-api/chapter-3.md)
```

Comment:
```suggestion
  * [Train & evaluate](guides/functional-api/chapter-3.md)
```

Reply:
Thanks, I have pushed the fixed version.